tomopy/io/exchange.py

```diff
@@ -83,7 +84,25 @@
            'read_petraIII_p05',
            'read_sls_tomcat']
 
-
+def deprecated(func):
```

Comment:
not sure ether. I could  not find any 'best practices' recommendation, but it looks reasonable.  
Below is how it will look like